salt/kube-apiserver/apiserver.jinja

```diff
@@ -23,7 +23,11 @@ KUBE_ETCD_SERVERS="--etcd-cafile={{ pillar['ssl']['ca_file'] }} \
 KUBE_SERVICE_ADDRESSES="--service-cluster-ip-range={{ pillar['services_cidr'] }}"
 
 # default admission control policies
+{% if salt.caasp_pillar.get('addons:psp', True) %}
 KUBE_ADMISSION_CONTROL="--admission-control=Initializers,NamespaceLifecycle,LimitRanger,ServiceAccount,NodeRestriction,PodSecurityPolicy,PersistentVolumeLabel,DefaultStorageClass,ResourceQuota,DefaultTolerationSeconds"
+{% else %}
+KUBE_ADMISSION_CONTROL="--admission-control=Initializers,NamespaceLifecycle,LimitRanger,ServiceAccount,NodeRestriction,PersistentVolumeLabel,DefaultStorageClass,ResourceQuota,DefaultTolerationSeconds"
```

Comment:
I think we can better have a real array for this, something like:

```
{% set kube_admission_control = ['Initializers', 'NamespaceLifecycle', 'LimitRanger', 'ServiceAccount', 'NodeRestriction', 'PersistentVolumeLabel', 'DefaultStorageClass', 'ResourceQuota', 'DefaultTolerationSeconds'] %}
{% if salt.caasp_pillar.get('addons:psp', True) %}
  {% do kube_admission_control.append('PodSecurityPolicy') %}
{% endif %}
KUBE_ADMISSION_CONTROL="{{ kube_admission_control|join('.') }}"
```

Reply:
i was just pushing an update when you comemnted, tried your suggestion afterwards on https://github.com/qn7o/jinja2-live-parser but it didn't work, i have it slightly different